.idea/misc.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
OK, I think I'm the one who suggested checking this stuff in, but I was wrong. You should add `.idea/` and `out/` to your .gitignore. There's an example template for JetBrains IDEs here: https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore